recipes/cpp-sort/all/conanfile.py

```diff
@@ -13,7 +14,7 @@
 class CppSortConan(ConanFile):
     name = "cpp-sort"
     description = "Additional sorting algorithms & related tools"
-    topics = "conan", "cpp-sort", "sorting", "algorithms"
+    topics = "cpp-sort", "sorting", "algorithms"
```

Comment:
We've also started to remove the name since it does not help the search

```suggestion
    topics = "sorting", "algorithms"
```